README.md

```diff
@@ -14,6 +14,12 @@ Rails 5.2
 $ rvm gemset create rails_5_2
 $ rvm gemset use rails_5_2
 $ gem install rails --pre
+$ gem devise
+$ gem cucumber-rails
+$ gem 'rspec-rails'
+$ gem 'shoulda-matchers'
+$ gem 'cucumber-rails', require: false
+$ gem 'launchy'
```

Comment:
I'm not sure if listing gems used in the app in conjunction with the commands used to create the app is a good move. Might be a bit confusing to the reader.